.pre-commit-config.yaml

```diff
@@ -15,6 +15,6 @@ repos:
     -   id: end-of-file-fixer
     -   id: debug-statements
 -   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.8.0a2
+    rev: 3.8.4
```

Comment:
Is there a way to have dependabot do this for us?

Reply:
good question!